src/domain/entities/Bridge.js

```diff
@@ -0,0 +1,34 @@
+class Bridge {
+  #spaces;
+  /**
+   *
+   * @param {Array<'U'|'D'>} spaces
+   */
+  constructor(spaces) {
+    this.#spaces = spaces;
+  }
+
+  /**
+   * return the spaces of the bridge from start to location
+   * @param {number} location
+   * @returns
+   */
+  getBridge(location) {
+    return [...this.#spaces].filter((space, index) => index <= location);
```

Comment:
space는 사용 안 하니까 아래처럼 할 수 있을 것 같아요!
```suggestion
    return [...this.#spaces].filter((_, index) => index <= location);
```

Reply:
아! 그렇네요! 이렇게 해줘야 안쓸거라는게 더 명확하겠네요!:)